pyspecdata/core.py

```diff
@@ -5466,7 +5474,10 @@ def extend(self,axis,extent, fill_with=0, tolerance=1e-5):
             stop_index_addto = int((extent - u[-1]) // du)
             if ((extent - u[-1]) - du * stop_index_addto)/du > tolerance:# the first quantity here is negative
                 stop_index_addto += 1
+            if stop_index_addto > 100:
+                stop_index_addto = 1
```

Comment:
why is there a limit here?

Reply:
I would reiterate this -- I don't understand why there's a limit here.

Reply:
This has been removed